### DISCUSSION TITLE
Support the deployment of 3D object detection algorithm SMOKE on Jetson 支持3D目标检测模型SMOKE在Jetson边缘计算设备部署

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#557] [MMDeploy]
# Support the deployment of 3D object detection algorithm SMOKE on Jetson 
 支持3D目标检测模型SMOKE在Jetson边缘计算设备部署 

Support the deployment of SMOKE algorithm from mmdetecion3d on Jetson device.
You can refer to the following resources:
1. [SMOKE]([ttps://github.com/open-mmlab/mmdetection3d/blob/main/configs/smoke/README.md](ttps://github.com/open-mmlab/mmdetection3d/blob/main/configs/smoke/README.md) )
2. [Install MMDeploy on Jetson]([https://mmdeploy.readthedocs.io/zh_CN/latest/01-how-to-build/jetsons.html](https://mmdeploy.readthedocs.io/zh_CN/latest/01-how-to-build/jetsons.html) )
3. [MMDet3d Deployment]([https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmdet3.html](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmdet3.html) )
4. [Deployment Tutorials]([https://mmdeploy.readthedocs.io/zh_CN/latest/tutorial/01_introduction_to_model_deployment.htm](https://mmdeploy.readthedocs.io/zh_CN/latest/tutorial/01_introduction_to_model_deployment.htm) )

在mmdeploy中支持SMOKE模型在Jetson设备上的部署
相关资源：
1. [https://github.com/open-mmlab/mmdetection3d/blob/main/configs/smoke/README.md](https://github.com/open-mmlab/mmdetection3d/blob/main/configs/smoke/README.md) 
2. [https://mmdeploy.readthedocs.io/zh_CN/latest/01-how-to-build/jetsons.html](https://mmdeploy.readthedocs.io/zh_CN/latest/01-how-to-build/jetsons.html) 
3. [https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmdet3d.html](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmdet3d.html) 
4. [https://mmdeploy.readthedocs.io/zh_CN/latest/tutorial/01_introduction_to_model_deployment.htm](https://mmdeploy.readthedocs.io/zh_CN/latest/tutorial/01_introduction_to_model_deployment.htm) 

## Task Information

**Repository**：MMDeploy

**Difficulty**：Hard

**Points**：60

**Estimated Time Consumption**：35 days

## Task Status

**Claimant's Github ID**：zhjunqin

**Claim Time**：2023年07月26日

**PR Link**：https://github.com/open-mmlab/mmdeploy/pull/2312

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by zhjunqin** [comment]
https://github.com/open-mmlab/mmdeploy/pull/2312  PR merged
